hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/StateNetworkInfo.java

```diff
@@ -89,6 +111,15 @@ public boolean containsNode(final long nodeId) {
 
     @Override
     public void updateFrom(@NonNull final State state) {
+        final var config = configProvider.getConfiguration();
+        if (config.getConfigData(TssConfig.class).keyCandidateRoster()) {
```

Comment:
We may want a different flag for this, or to rename this flag–`keyCandidateRoster` has a much more specific meaning. This code is essentially just checking if TSS processing is on or off, right?

Reply:
yeah there is `enableLedgerId` also. This flag is used as feature flag because if `setCandidateRoster` is not called, none of the TSS logic is triggered

Reply:
TSS-Roster is setting a candidate roster for a different life-cycle, instead of writing config.txt to disk.  I'm wondering about collision between the two different life-cycles.